index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
@w0rm Ok I will test with my postcss test env and project using postcssrc (PoC)

```
npm i postcss/gulp-postcss#config
```

right ?